jujugui/static/gui/src/app/jujulib/test-rates.js

```diff
@@ -0,0 +1,40 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+
+'use strict';
+
+describe('jujulib rates service', () => {
+
+  it('exists', () => {
+    const webHandler = {};
+    const rates = new window.jujulib.rates('http://1.2.3.4/', webHandler);
+    assert.equal(rates.url, 'http://1.2.3.4/' + window.jujulib.ratesAPIVersion);
```

Comment:
If ratesAPIVersion is only exported for testing, I'd rather repeat 'v3' literal here.

Reply:
done